igvc_navigation/launch/octomap.yaml

```diff
@@ -0,0 +1,72 @@
+octree:
+  resolution: 0.2
+  lazy_evaluation: false
+  clamping:
+      max: 1.0
+      min: 0.0
+probability_model:
+    lidar:
+        scan:
+          prob_miss: 0.3
+          prob_hit: 0.9
+        ground:
+          prob_miss: 0.4
+          prob_hit: 0.6 # Not used
+        free_space:
+          prob_miss: 0.2
+          prob_hit: 0.6 # Not used
+    camera:
+        prob_miss: 0.45
+        prob_hit: 0.9
+sensor_model:
+    max_range: 3
```

Comment:
I'm pretty sure for parameters like this it should be fine to leave out the comments, since the units are defined by REP 103, and I feel like adding in comments to every field that has a unit and is trivial clutters up the file a lot.